src/components/InputTypeCreditCard.js

```diff
@@ -24,8 +26,10 @@ class InputTypeCreditCard extends React.Component {
     this.props.onChange({ [fieldname]: value });
   }
 
-  componentDidMount() {
-    if (typeof window.stripe !== 'undefined') {
+  async componentDidMount() {
+    const stripe = await getStripe();
+
+    if (stripe) {
```

Comment:
if stripe is not defined, this will never get called again (`componentDidMount` is only called once) and we might have an issue no?
It would be better to return an error if `stripe` is not defined (and track it)

Reply:
The matching PR is:
https://github.com/opencollective/opencollective-frontend/pull/836

I see 2 failing cases, one is the STRIPE_KEY missing, the other is a potential network error while loading the stripe library. We can definitely throw an exception.